frontend/main-app/src/components/FormTextfield/FormTextfield.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Heading, Textfield } from "@digdir/design-system-react";
+
+interface FormTextFieldProps {
+  name: string;
+  question: string;
+  error?: string[];
+  required: boolean;
+}
+
+export const FormTextfield: React.FC<FormTextFieldProps> = ({ question, error, name, required }) => {
+  const requiredPlaceholder = required ? "Required" : undefined;
+  return (
+    <>
+      <Heading level={6} size="xxsmall" spacing>
+        {question}
+      </Heading>
+      <Textfield name={name} error={error} placeholder={requiredPlaceholder} />
+    </>
```

Comment:
```suggestion
      <Textfield name={name} error={error} placeholder={requiredPlaceholder} label={question} />
    </>
```

Reply:
Perhaps it would be more beneficial to use a label prop in the `textfield` component instead of adding a `Heading`. This way, you can establish the `htmlFor` connection between the `label` and `textfield`, which I believe is more semantically correct in this case.

Reply:
Of course - nice catch!